Sources/DarkMode.swift

```diff
@@ -1,11 +1,16 @@
 import Foundation
+import AppKit
 
 struct DarkMode {
 	private static let prefix = "tell application \"System Events\" to tell appearance preferences to"
 
 	static var isEnabled: Bool {
 		get {
-			UserDefaults.standard.string(forKey: "AppleInterfaceStyle") == "Dark"
+			if #available(macOS 10.14, *) {
+				return NSAppearance.current.name == .darkAqua
```

Comment:
This is not correct. It doesn't take into accessibility settings.

```suggestion
				return NSAppearance.current.bestMatch(from: [.darkAqua, .aqua]) == .darkAqua
```

Reply:
Great catch, reading the docs I managed to scan over the accessibility lines thinking they where also related to visual effects (as vibrantLight and vibrantDark is). Will incorporate your suggested change.